api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/reshaping/ZeroPadding2D.kt

```diff
@@ -20,9 +18,8 @@ import org.tensorflow.op.Ops
  *
  * @property [padding] 4 numbers  interpreted as `(top_pad, bottom_pad, left_pad, right_pad)`.
  *
- * @see [Tensorflow implementation](https://github.com/tensorflow/tensorflow/blob/582c8d236cb079023657287c318ff26adb239002/tensorflow/python/keras/layers/convolutional.py#L2765)
  */
-public class ZeroPadding2D : Layer {
+public class ZeroPadding2D : AbstractZeroPadding {
     public val padding: IntArray
     private val dataFormat: String
     private lateinit var inputShape: Shape // TODO: refactor this shape to another property in another place
```

Comment:
Hey @zaleslaw  should i also remove this TODO from ZeroPadding2D?

Reply:
Please remove everywhere in this PR TODOs related to the order of the channels NWHC. This TODO looks unclear for me, explain here better or remove it.